cmd/client/txgen/main.go

```diff
@@ -123,6 +123,8 @@ func main() {
 	consensusObj := consensus.New(host, 0, nil, p2p.Peer{}, nil)
 	clientNode := node.New(host, consensusObj, nil, false)
 	clientNode.Client = client.NewClient(clientNode.GetHost(), shardIDs)
+	consensusObj.SetStakeInfoFinder(clientNode)
```

Comment:
It won't work with txgen, why do you need to change it here?

Reply:
Once txgen is fixed to deal with staking, the call will become necessary here.  But for now I'll back it out so it doesn't get into others' merges.

Reply:
Fixed in b7df7eec4de725f935b1cc495c228ad9d59670f2.